### ISSUE TITLE
Browser not reloading

### ISSUE THREAD

**Issue by aaronmeder** [hirasso/parcel-plugin-watch-reload#6]
Plugin looks great!

I have enabled it for PHP files and it detects the file changes on the command line:
`wp-content/themes/limpido/footer.php changed: reloading browsers...`

...However it doesn't reload the Browser.
A manual refresh shows the code changes.

Can you help? It would be useful!

**Some Background Info**
Using parcel to build JS and SCSS in a Wordpress theme.

package.json
`{
  "name": "XYZ",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "dev": "parcel watch ./wp-content/themes/limpido/src/parcel-entrypoint.js --out-file=site --out-dir=wp-content/themes/limpido/dist",
    "build": "parcel build ./wp-content/themes/limpido/src/parcel-entrypoint.js --out-file=site --out-dir=wp-content/themes/limpido/dist"
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "autoprefixer": "^9.7.1",
    "parcel-bundler": "^1.12.4",
    "parcel-plugin-watch-reload": "0.0.2",
    "postcss-modules": "^1.4.1",
    "sass": "^1.23.3",
    "spectre.css": "^0.5.8"
  },
  "watchreload": {
    "files": "**/*.php"
  }
}
`

.postcssrc
`{
  "modules": false,
  "plugins": {
    "autoprefixer": {
      "grid": true
    }
  }
}`

**Comment by hirasso** [comment]
Hi @aaronmeder , you have to put this inside your main js entry file:
```javascript
if( module.hot ) {
  module.hot.accept(() => {
    window.location.reload();
  })
}
```
...this will disable Hot Module Reloading, but there is currently no way I know of how to ONLY reload the site if the change was coming from outside of parcel.

**Comment by aaronmeder** [comment]
Hey @hirasso 

I just found my issue was unrelated to your plugin.

Have moved the local dev site to an SSL connection and it seems like the websocket connection Parcel calls for HMR was not happy with the self signed certificate. So the HMR was totally broken, not only for use with this plugin.
 
After I have switches back to an unencrypted connection it worked right away - both normal Parcel HMR and auto reloading based on changes made in PHP files using your plugin. I did not need the change you suggested in your recent answer.

Thanks a lot for developing the plugin - makes development an even bigger joy 🎉
Aaron
